.github/workflows/central-publish.yml

```diff
@@ -61,6 +61,7 @@ jobs:
           packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
         run: |
+          sed -i 's/version=\(.*\)-SNAPSHOT/version=\1/g' gradle.properties
```

Comment:
Do we need this in the central publish workflow?

Reply:
This is only added for dev_central and staging_central steps to avoid this failure: https://github.com/ballerina-platform/module-ballerina-oauth2/actions/runs/4361652636/jobs/7625738945#step:10:212. This is not added to the central step